yarn.lock

```diff
@@ -5,6 +5,7 @@
 "@babel/code-frame@^7.0.0":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
+  integrity sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==
```

Comment:
We have the project yarn specified as 1.9.2
https://github.com/all-contributors/all-contributors-bot/blob/master/.yvmrc

We should bump this version if we want to use a newer yarn version (so everyone contributing to the project is on the same version!)

Reply:
Yeah, so `1.13.0`?

Reply:
Sure, I'll downgrade it for this branch, push the changes and create a PR with `yarn@1.13.0`.